ui/src/pages/namespace/Explorer/Workflow/Overview/Metrics.tsx

```diff
@@ -65,7 +37,7 @@ const Metrics = ({ workflow }: { workflow: string }) => {
         <MetricsRefetchButton />
       </div>
       {metrics ? (
```

Comment:
I found two little issues here. The behavior of the API has changed to always return a result like this (even when the path does not exist, but that is not important here... )

```JSON
{
  "data": {
    "cancelled": 0,
    "complete": 0,
    "crashed": 0,
    "failed": 0,
    "pending": 0,
    "total": 0
  }
}
``` 

So metrics is only `undefined` when it is in the loading state. So the "no results" message is displayed for a very short time when the page is loading (this "bug" was probably in the version before as well, maybe we simply add an early return?) and never when workflow has not been executed yet (this is new). 


The second issue resolves in a dividing by zero problem:

![CleanShot 2024-04-23 at 13 42 28@2x](https://github.com/direktiv/direktiv/assets/121789579/6fa007b8-7c04-4e04-b993-29a94737808f)

Reply:
Solved in a68e38ce6. I didn't use an early return, but I leave "metrics" undefined until the fetch request has finished (avoiding the NaN output). I also added a "loading..." placeholder. I left the "no results" placeholder in the component. It may be dead code if the API always returns an object, but it makes sense to leave it in in case this is fixed.